crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -5,22 +5,46 @@
 def read_names():
     """
     Read the names in ../names
-    :return: names_list
+    :return: list of names
     """
-    names_list = []
-    for name_file in glob.glob("./names/*"):
+    names = []
+    for name_file in glob.glob("../names/*"):
         with open(name_file, 'r') as f:
             first_line = f.readline()
             first_name = first_line.split()[0]
-            names_list.append(first_name)
-    return names_list
+            names.append(first_name)
+    return names
 
 
-def greet(names):
-    greeting_list = ["Hello", "Hi", "Howdy","Hola","Ciao"]
-    print("%s, %s!" % (random.choice(greeting_list),
-                       random.choice(names)))
+greetings = ["Hello", "Hi", "Howdy", "Hola", "Ciao", "G'day", "What's up", "Howzgarn"]
```

Comment:
this is really not a "global variable" but a constant.  Pep8 tells us to make the name all in caps
`GREETINGS`